src/Serialization/Address.js

```diff
@@ -47,7 +47,14 @@ exports.addressBytes = callToBytes;
 exports.byronAddressBytes = callToBytes;
 exports.stakeCredentialToBytes = callToBytes;
 
-exports.addressBech32 = callToBech32;
+exports.addressBech32 = addr => {
+    // 0 TestnetId
+    // 1 MainnetId
+    return addr.network_id() == 0 ? addr.to_bech32("addr_test") : addr.to_bech32("addr");
```

Comment:
Wait I was wrong, we actually know the network id. Then we can print the address in a format acceptable by explorers. Could you please use this function?

Reply:
Im having troubles running examples and tests. Meanwhile discovered that `to_bech32` called parameterless should prepend the right prefix:
```
    pub fn to_bech32(&self, prefix: Option<String>) -> Result<String, JsError> {
        let final_prefix = match prefix {
            Some(prefix) => prefix,
            None => {
                // see CIP5 for bech32 prefix rules
                let prefix_header = match &self.0 {
                    AddrType::Reward(_) => "stake",
                    _ => "addr",
                };
                let prefix_tail = match self.network_id()? {
                    id if id == NetworkInfo::testnet().network_id() => "_test",
                    _ => "",
                };
                format!("{}{}", prefix_header, prefix_tail)
            }
        };
```

Reply:
Will try to verify it. But that may end up waiting for tomorrow. Not sure what is your request exactly.

Reply:
It looks like the entire issue was created because of my misunderstanding (I wasn't aware that it should be OK).

It all started from this message: https://discord.com/channels/922576618424250418/940666072208662628/991274629983391786

Reply:
Could you please check that common testnet explorer indeed parse the addresses in this format, just to be sure?

Reply:
> Im having troubles running examples and tests.
It also takes place on `develop` branch.